tests/sqlparser_mysql.rs

```diff
@@ -1035,6 +1036,92 @@ fn parse_ignore_insert() {
     }
 }
 
+#[test]
```

Comment:
Can you please add test coveage for LOW_PRIORITY and any errors (for example, unsupported `REPLACE`)

Reply:
Added extra tests